Gemfile.lock

```diff
@@ -73,16 +74,32 @@ GEM
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     parallel (1.19.1)
+    parser (2.7.1.3)
+      ast (~> 2.4.0)
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
+    rainbow (3.0.0)
     rake (13.0.1)
     rb-fsevent (0.10.4)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
     ref (2.0.0)
+    regexp_parser (1.7.0)
```

Comment:
This doesn't seem to be a dependency for `rubocop`, could you please elaborate?

Reply:
`s.add_runtime_dependency('regexp_parser', '>= 1.7')` is on line 39 of the gemspec for Rubocop.